tasks/push-disk-images/push-disk-images.yaml

```diff
@@ -86,6 +94,7 @@ spec:
         results=$(kubectl get internalrequest "$internalRequest" -o=jsonpath='{.status.results}')
         if [ "$(echo "${results}" | jq -r '.result')" == "Success" ]; then
           echo "Disk images pushed"
+          echo "${results}" | jq -r '.'
```

Comment:
This is a structured json (not just one value), so `-r` doesn't make sense here - it only applies for single strings. If you just want to format it, all you need is `jq`, nothing else.

Reply:
fixed. thanks

Reply:
I meant you don't even need the `'.'`, but ok :)